src/sinks/vector.rs

```diff
@@ -75,10 +74,6 @@ fn encode_event(event: Event) -> Option<Bytes> {
     let event_len = event.encoded_len();
     let full_len = event_len + 4;
 
-    emit!(VectorEventSent {
-        byte_size: full_len
-    });
-
```

Comment:
We might want to convert this one to a sink-specific emit/metric to preserve the bytes sent data point.

Reply:
We have that data point emitted in `BytesSink` which is being used by `TcpSink` which is in turn being used by this sink.
https://github.com/timberio/vector/blob/ae18f8f7790c56bd916cc634a0cb818d1e6db9c4/src/sinks/util/socket_bytes_sink.rs#L70-L74

Or do you mean some other data point?